packages/rocketchat-ui/client/views/app/createChannel.html

```diff
@@ -1,90 +1,98 @@
 <template name="createChannel">
 	<section class="create-channel">
 		{{#each roomType in roomTypesBeforeStandard}}
-			{{> Template.dynamic template=roomType.creationTemplate }}
+			<div class="room-type-creation">
+				{{> Template.dynamic template=roomType.creationTemplate }}
+			</div>
 		{{/each}}
-		<header class="create-channel__header">
-			<h1 class="create-channel__title">{{_ "Create_A_New_Channel"}}</h1>
-			<p class="create-channel__description">{{_ "Channels_are_where_your_team_communicate"}}</p>
-		</header>
-		<form class="create-channel__content">
-			<div class="create-channel__switches">
-				<div class="rc-switch">
-					<label class="rc-switch__label" tabindex="-1">
-						<input type="checkbox" class="rc-switch__input" name="type" value="p" checked>
-						<span class="rc-switch__button">
+		<div class="room-type-creation">
+			<header class="create-channel__header">
+				<h1 class="create-channel__title">{{_ "Create_A_New_Channel"}}</h1>
```

Comment:
`.create-channel__header .create-channel__title` as child of `.room-type-creation` could be the title of the tab